.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,6 +9,8 @@
   - Mention the GitHub issue that is being addressed by the pull request.
   - The keywords `Fixes`, `Closes`, or `Resolves` followed the issue number will automatically close the issue.
 
+> NOTE: All non-trivial PRs should have an associated issue, and any new features or components must reference an I2I (intent-to-implement: https://github.com/ampproject/amphtml/issues/new?assignees=&labels=INTENT+TO+IMPLEMENT&template=intent-to-implement--i2i-.md&title=I2I:%20%3Cyour%20change/update%3E). Please read through the contribution process (https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md) for more information.
```

Comment:
- `non-trivial` might be unclear since the definition is largely due to our perception of scope size. Maybe adding an example like `non-trivial changes (like introducing new features or components)`.

- Those URLs are rough, [maybe we can add golinks](https://github.com/ampproject/amp.dev/blob/future/platform/config/go-links.yaml)?

Reply:
Updated the copy re: non-trivial, lmk what you think.

Re: go-links, I totally agree. I created https://github.com/ampproject/amp.dev/pull/3254 , will update these links if/when that's approved

Reply:
Wording LGTM and thanks for adding the golinks!

Reply:
Updated the links here to use go links, assuming the `amp.dev` PR goes in as it is now.

Reply:
Nice!

Reply:
The go links are in! Do you know how long I need to wait before they are active/how frequently amp.dev deploys?